pages/posts/github-case-study.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "How JSON Schema Was an Obvious Choice at GitHub"
+date: 2023-07-31
+type: Case Study
+cover: /img/posts/2023/github-case-study/github-mark.svg
+authors:
+  - name: Julian Berman
+    photo: /img/avatars/julian.webp
+    link: https://mastodon.social/@JulianWasTaken
+    byline: JSON Schema Test Suite Maintainer & Technical Lead, API Specifications @Postman
+excerpt: "At GitHub's Docs Engineering team, while shipping releases to production 20 times per day or more, JSON Schema is critical in increasing confidence in changes to data, content and APIs."
+---
+
+## Challenge
+
+GitHub stands as a ubiquitous and invaluable tool for software developers and those in related fields, enabling efficient code management, seamless collaboration, automation, project management and more.
+More than 100 million users rely on the platform, many of them making use of it daily for building and sharing open source software.
+
+There is a staggering amount of functionality packed into GitHub, assisting across all parts of the development lifecycle, and supporting this ever-growing list of functionality is an undoubtedly powerful and complex underlying platform developed within GitHub, as well as extensive documentation covering features new and old.
+
+Documentation at GitHub is housed at [docs.github.com](https://docs.github.com), itself a massive undertaking, covering pages upon pages of functionality within the platform.
+Historically, this documentation was split into two statically generated sites (help.github.com and developer.github.com), but in 2020 these static sites were combined in a new dynamic application hosted at docs.github.com.
+The full story of the rebirth of GitHub's documentation is told best on the GitHub blog [here](https://github.blog/2020-10-14-how-we-open-sourced-docs-github-com/) and [here](https://github.blog/2020-07-02-how-we-launched-docs-github-com/), but an important outcome of this change was a move towards data-driven documentation, and in particular a big investment in the use of JSON both to drive content as well as for intercommunication within the application's platform itself.
+
+More specifically, some of page content is assembled fully automatically from JSON data files and some JSON is manually authored by content writers.
```

Comment:
I think this might be a typo.

```suggestion
More specifically, some page content is assembled fully automatically from JSON data files and some JSON is manually authored by content writers.
```

Reply:
Indeed, thanks!!